ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -345,6 +351,70 @@ private void onAccountChangeMessage(String topic, String message) {
     }
   }
 
+  /**
+   * Selects {@link Container}s to be marked as INACTIVE. Check the valid data size of each DELETE_IN_PROGRESS container
+   * from {@link StatsSnapshot} and select the ones with zero data size to be marked as INACTIVE.
+   */
+  Set<Container> selectInvalidContainers(StatsSnapshot statsSnapshot) {
```

Comment:
should we use `Deprecated` for naming consistency?

Reply:
I've updated the method name from selectInvalidContainers to selectInvalidContainerCandidates which stands for select all containers which will be marked as Invalid during this aggregation cycle.

Reply:
OK. It means you select DELETE_IN_PROGRESS and mark it as INACTIVE in the end.